### PR TITLE
Added contribute other people repositories section

### PR DESCRIPTION
**Pull request by diegogsann**
Related to #268 

@RichardLitt can you suggest enhancements here? we can do better together.
thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1767,6 +1767,76 @@ function Squash-Commits {
 }
 ```
 
+# Adding code to other people's repositories.
```
Comment (RichardLitt) [inline README.md]: Needs to be added to the TOC. I'm also not sure that this should be a top-level entry.

File: README.md [hunk 2]
```diff
@@ -1767,6 +1767,76 @@ function Squash-Commits {
...
+
+Unless you are a contributor, you can't do it directly. But you can suggest code using forks and pull requests.
```
Comment (RichardLitt) [inline README.md]: Your entry is a bit GitHub-specific - you can also use patches.

File: README.md [hunk 3]
```diff
@@ -1767,6 +1767,76 @@ function Squash-Commits {
...
+
+## Forking
+A fork is a copy of repository. It is not a git operation, but is a common action on GitHub, Bitbucket, GitLab — or anywhere people host Git repositories.
```
Comment (RichardLitt) [inline README.md]: Git should be capitalized.

File: README.md [hunk 4]
```diff
@@ -1767,6 +1767,76 @@ function Squash-Commits {
...
+
+## Suggesting code via pull requests
+
+After forked, you need to clone repository to your machine.
```
Comment (RichardLitt) [inline README.md]: Describe how you fork a project.

File: README.md [hunk 5]
```diff
@@ -1767,6 +1767,76 @@ function Squash-Commits {
...
+
+```sh
+# if you are using ssh
+$ git clone git@github.com:k88hudson/git-flight-rules.git
+
+# if you are using https
+$ git clone https://github.com/k88hudson/git-flight-rules.git
+```
+
+Set up a new remote that points to the original project. Generally we use `upstream` as a remote name.
```
Comment (RichardLitt) [inline README.md]: Generally > "Usually, we use"

File: README.md [hunk 6]
```diff
@@ -1767,6 +1767,76 @@ function Squash-Commits {
...
+
+## Suggesting code via pull requests
+
+After forked, you need to clone repository to your machine.
+
+```sh
+# if you are using ssh
+$ git clone git@github.com:k88hudson/git-flight-rules.git
+
+# if you are using https
```
Comment (RichardLitt) [inline README.md]: Can you describe the difference?

File: README.md [hunk 7]
```diff
@@ -1767,6 +1767,76 @@ function Squash-Commits {
...
+```
+
+Set up a new remote that points to the original project. Generally we use `upstream` as a remote name.
+```sh
+$ (master) git remote add upstream git@github.com:k88hudson/git-flight-rules.git
+```
+
+Note that now you have two remotes.
+
+- `origin` are referencing your own repository.
```
Comment (RichardLitt) [inline README.md]: are referencing > references

**Comment by diegogsann** [comment]
> Generally this looks fine. There are some grammar issues here and there - mind if I PR your PR?

Sure! Will be great :)
thanks.

**Comment by RichardLitt** [comment]
I wasn't able to PR, as your commits come from an unknown repo. I've opened a new PR in #274.

**Pull request closed without merging**